Sources/Widgets/Representations/WidgetRepresentation/index.js

```diff
@@ -221,7 +221,7 @@ function vtkWidgetRepresentation(publicAPI, model) {
     }
 
     const rHeight = rendererPixelDims[1];
-    return scale / rHeight;
+    return (window.devicePixelRatio * scale) / rHeight;
```

Comment:
are parenthesis needed ?

Reply:
Also that assume the vtkRenderWindow use `window.devicePixelRatio` to scale up its resolution.